src/utils/api.ts

```diff
@@ -0,0 +1,24 @@
+import { Post } from '../types/Post';
+import { User } from '../types/User';
+import { Comment } from '../types/Comment';
+import { client } from './fetchClient';
+
+export const getUsers = () => {
+  return client.get<User[]>('/users');
+};
+
+export const getPosts = (userId: number) => {
+  return client.get<Post[]>(`/posts?userId=${userId}`);
+};
+
+export const getComment = (postId: null | number) => {
+  return client.get<Comment[]>(`/comments?postId=${postId}`);
+};
+
+export const getDeleteComment = (postId: null | number) => {
```

Comment:
The function name 'getDeleteComment' does not reflect its operation. This function is used to delete a comment, so it would be better to rename it to 'deleteComment'.